linera-views/src/test_utils/mod.rs

```diff
@@ -9,176 +9,6 @@ use crate::batch::{
 use rand::{Rng, RngCore};
 use std::collections::HashSet;
 
-#[cfg(feature = "aws")]
```

Comment:
(see previous comment) Is moving this code to the main module what we want?

Reply:
Yes, this is certainly what I want.
For the code for RocksDb, ScyllaDb, and DynamoDb, there is great commonality due to their use of a common database. That commonality overweights some other commonality such that being part of test.

Reply:
As discussed, I'm ok with it as long as the code is still gated by `#[cfg(any(test, feature = "test"))]`.